### DISCUSSION TITLE
No docs on route handling

### DISCUSSION THREAD

**Discussion by joauea** [jaredhanson/passport#823] [Help]
This is, I'll admit, partially a request for help. But there's a general point as well: the Passport docs simply don't explain how it interacts with route handling. I believe it's causing the issue I posted here: https://stackoverflow.com/questions/57470000/cant-get-passport-js-redirect-and-flash-message-to-work (although correct me if I'm wrong). If so, the questions are:

1. How to use Passport with multiple route files?
2. Are the paths in e.g., `failureRedirect` relative to the application root or to the particular endpoint on which they're defined? This is particularly confusing if you're using a multiple route-file setup. (Because if relative to the endpoint, you'd have to do something like `'../../'` to get back to the application root. Although that doesn't seem to work in my case above.)

**Comment by madacol** [comment]
Are there docs? :expressionless:

**Comment by ghost** [comment]
Sorry to necro bump, but incase you wanted to resolve this issue. Please share your findings if you were able to resolve this but if not.

Can you share some version of your package list if not a complete repo or simple gist? I would need to know what frontend you're concerned with, what server you're using, and what your goal is, to put myself in your shoes. Your questions do make sense but I'm just stepping into this package today.
